src/tools/spark.md

```diff
@@ -123,6 +140,7 @@ and is typically suitable for reading smaller amounts of data. If you need
 to read large amounts of data, prefer the Storage API as described above.
 
 Example:
+
 ```python
 
```

Comment:
```suggestion
```
Remove empty line

Reply:
I think the extra line helps with readability (when viewing it as markdown), so am inclined to keep it.

Reply:
This way it's not consistent with other code blocks in the same file.

Reply:
Ah ok, I'll do a quick followup